homeassistant/components/sensor/ambient_station.py

```diff
@@ -5,21 +5,21 @@
 https://home-assistant.io/components/sensor.ambstation/
 """
 
-import logging
-import time
 from datetime import timedelta
+import logging
+import asyncio
```

Comment:
🔡

Reply:
I re-ran isort and it sorted imports as:

`import asyncio
from datetime import timedelta
import logging`

Reply:
That's ok.